lib/enkf/obs_data.cpp

```diff
@@ -760,3 +771,16 @@ void obs_data_fprintf( const obs_data_type * obs_data , FILE * stream) {
   }
   fprintf(stream , "\n");
 }
+
+
+const bool * obs_data_get_active_mask( const obs_data_type * obs_data ) {
+  int total_size = obs_data_get_total_size( obs_data );
```

Comment:
At the site where this will be used we will need size of mask - so although it is on the way out; I want this to be implemented with `bool_vector_type`.

Furthermore the `obs_data`structure must hold on to the `obs_mask`, and free it when the `obs_data` is discarded. It should not be the responsability of the calling scope to free the mask.